evoKnowledgeShareFrontend/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,12 @@
+import React, {useState} from 'react'
+import IDocument from '../interfaces/IDocument';
+
+export default function useLocalStorage(props:IDocument){
+    let [elements, setElements] = useState(()=> {
+        
+    })
+
+
+
+    return[elements, setElements];
+}
```

Comment:
lehet csak én nem látom, de ennek a metódusnak mi az értelme?

Reply:
Ezzel majd azt szeretném csinálni, hogy nem state-be mentem el a md fájl tartalmát hanem localstorage-be és ha félbe hagyja a szerkesztést akkor is tudja folytatni később de most még sehol sincs ez használva